### PR TITLE
Enroll ZKV/Staked to default ism

### PR DESCRIPTION
**Pull request by nambrot**
### Description

Enroll ZKV/Staked to default ism

### Drive-by changes

Fix verify-validators script

### CODE REVIEW

File: typescript/infra/config/environments/mainnet2/core.ts [hunk 1]
```diff
@@ -103,6 +117,7 @@ export const core: ChainMap<MainnetChains, CoreConfig> = {
         '0x02424d4222f35c04da62a2f2dea8c778030bb324',
         '0x9509c8cf0a06955f27342262af501b74874e98fb',
         '0xb7113c999e4d587b162dd1a28c73f3f51c6bdcdc',
+        '0x26725501597d47352a23cd26f122709f69ad53bc', // staked
```
Comment (asaj) [inline typescript/infra/config/environments/mainnet2/core.ts:120]: Similarly, maybe 3/4 here instead?

File: typescript/infra/config/environments/mainnet2/core.ts [hunk 2]
```diff
@@ -13,6 +13,8 @@ export const core: ChainMap<MainnetChains, CoreConfig> = {
         '0x3bc014bafa43f93d534aed34f750997cdffcf007',
         '0xd79d506d741fa735938f7b7847a926e34a6fe6b0',
         '0xe4a258bc61e65914c2a477b2a8a433ab4ebdf44b',
+        '0x6aea63b0be4679c1385c26a92a3ff8aa6a8379f2', // staked
```
Comment (asaj) [inline typescript/infra/config/environments/mainnet2/core.ts:16]: Do you think a 3/7 is the right config? Maybe instead a 4/6 and remove an Abacus Works validator?

Comment (nambrot) [reply typescript/infra/config/environments/mainnet2/core.ts:15]: Indeed

File: typescript/infra/config/environments/mainnet2/core.ts (hunk 1 above)
Comment (nambrot) [reply typescript/infra/config/environments/mainnet2/core.ts:112]: yup

**Review by asaj — APPROVED** [review]

**Pull request merged**